vivisect/renderers/__init__.py

```diff
@@ -231,13 +226,20 @@ def renderLocation(self, mcanv, loc, name, isfunc, cmnt, extra):
         elif ltype == LOC_UNDEF:
 
             mcanv.addText(linepre, vatag)
-            offset, bytes = self.vw.getByteDef(lva)
-            b = bytes[offset:offset+1]
-            mcanv.addNameText(binascii.hexlify(b).decode('utf-8'), typename="undefined")
-            if b in string.printable.encode('utf-8'):
+            offset, bytez = self.vw.getByteDef(lva)
+            b = bytez[offset:offset+1]
+            mcanv.addNameText(b.hex(), typename="undefined")
+
+            try:
+                b = b.decode('utf-8')
+            except:
+                b = b.hex()
```

Comment:
woops.  i'm not sure i like this...  damn, i suppose i should have thought of that before merging it in ;)